sklearn/tests/test_docstrings.py

```diff
@@ -71,7 +71,10 @@ def filter_errors(errors, method, Klass=None):
 
         # Ignore PR02: Unknown parameters for properties. We sometimes use
         # properties for ducktyping, i.e. SGDClassifier.predict_proba
-        if code == "PR02" and Klass is not None and method is not None:
+        # Ignore GL08: Parsing of the method signature failed, possibly because this is
+        # a property. Properties are sometimes used for deprecated attributes and the
+        # attribute is already documented in the class docstring.
```

Comment:
Thanks for adding this comment.

I guess we could reference [the list of error codes](https://numpydoc.readthedocs.io/en/latest/validation.html#built-in-validation-checks) somewhere (e.g. in `filter_errors`' docstring).